.env.example

```diff
@@ -1,2 +1,4 @@
 VUE_APP_INFURA_KEY=
-VUE_APP_IS_MAINNET=false
\ No newline at end of file
+VUE_APP_OFFLINE_DB=tknbrgoff
+VUE_APP_SIDE_CHAIN_ID=1 # If not set, uses 1 as default
```

Comment:
maybe `VUE_APP_DEFAULT_SIDE_CHAIN_ID` and skip the comment?

Reply:
i think that VUE_APP_DEFAULT_SIDE_CHAIN_ID sounds like im setting the default and i can change that, i would just leave it as VUE_APP_SIDE_CHAIN_ID and remove the comment

Reply:
Do you mean create a new env variable as `VUE_APP_DEFAULT_SIDE_CHAIN_ID` or rename from VUE_APP_SIDE_CHAIN_ID to VUE_APP_DEFAULT_SIDE_CHAIN_ID